### PR TITLE
ok test yosi grando season

### PR DESCRIPTION
**Pull request by JooYongLee**
--

### CODE REVIEW

File: testFolder/sometest.py [hunk 1]
```diff
@@ -1,8 +1,20 @@
...
-    print('abc')
\ No newline at end of file
+    print('abc')
+
+
+def main():
+    print(np.random.randn(10))
+
+main()
+
```
Comment (yongyongnim) [inline testFolder/sometest.py:13]: ok merge good

File: testFolder/sometest.py [hunk 2]
```diff
@@ -1,8 +1,20 @@
...
+def test_fun(x):
+    y = 2 * x
+    return y
+
+test_fun()
+    #
+    # print('abcdef')
```
Comment (yongyongnim) [inline testFolder/sometest.py:20]: some good

**Review by yongyongnim — CHANGES_REQUESTED** [review]

File: testFolder/sometest.py (hunk 2 above)
Comment (JooYongLee) [reply testFolder/sometest.py:20]: ok

File: testFolder/sometest.py [hunk 3]
```diff
@@ -1,8 +1,20 @@
...
 print(np.random.randn(10))
 def test():
-    print('abc')
\ No newline at end of file
+    print('abc')
+
+
+def main():
+    print(np.random.randn(10))
+
```
Comment (yongyongnim) [inline testFolder/sometest.py:11]: okok

**Review by yongyongnim — COMMENTED** [review]
ok

**Pull request merged**